themes/classic/templates/errors/not-found.tpl

```diff
@@ -24,9 +24,17 @@
  *}
 <section id="content" class="page-content page-not-found">
   {block name='page_content'}
+    {if isset($title)}
+      <h4>{block name="searchTitle"}{$title}{/block}</h4>
+    {else}
+      <h4>{block name="searchTitle"}{l s='No matches were found for your search' d='Shop.Theme.Catalog'}{/block}</h4>
+    {/if}
```

Comment:
@NeOMakinG Can we do it like this? The block can be present only once, and renamed for better readability. 👍 

```suggestion
    {block name="notFoundTitle"}
      {if isset($title)}
        <h4>{$title}</h4>
      {else}
        <h4>{l s='No matches were found for your search' d='Shop.Theme.Catalog'}</h4>
      {/if}
    {/block}
```

Reply:
We also need to modify this template then : themes/classic/templates/errors/404.tpl

Reply:
@NeOMakinG And what if you do it like this??

    {block name="notFoundContent"}
      {if isset($notfoundcontent)}
        {$notfoundcontent}
      {else}
        <h4>{l s='No matches were found for your search' d='Shop.Theme.Catalog'}</h4>
        <p>{l s='Please try other keywords to describe what you are looking for.' d='Shop.Theme.Catalog'}</p>
      {/if}
    {/block}

Reply:
Then

```
{capture assign="notFoundContent"}
  <h4>{l s='This page could not be found' d='Shop.Theme.Global'}</h4>
  <p>{l s='Try to search our catalog, you may find what you are looking for!' d='Shop.Theme.Global'}</p>
{/capture}
```